lib/processor/index.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const UploadedFile = require('../uploaded-file');
+const logger = require('../logger');
+
+// aliases
+exports.download = require('./download');
+exports.validate = require('./validate');
+exports.uploadAndCopy = require('./upload');
+
+// do all the things
+exports.work = (audioEvent) => {
+  let file = new UploadedFile(audioEvent);
+
+  // process file - catch most errors
+  return exports.download(audioEvent.audioUploadPath).then(data => {
+    file.setDownloaded(data);
+    return Q.all([
+      exports.validate(file.localPath),
+      exports.uploadAndCopy(file)
+    ]);
+  }).then(results => {
+    file.setValidated(results[0]);
+    file.setProcessed(results[1]);
+    return file.callback().then(() => true);
+  }).catch(err => {
+    if (err.noRetry) {
+      logger.warn(err.message);
```

Comment:
any reason to be more explicit here in the logs and say "not retrying" along with error message?

Reply:
I figured this was the only `[WARN]` at the moment.  But yeah - I should probably make that more explicit.  Will do!